Mathlib/Topology/ContinuousMap/Bounded/Basic.lean

```diff
@@ -531,6 +531,36 @@ instance instCommMonoid [CommMonoid R] [BoundedMul R] [ContinuousMul R] :
   __ := instMonoid
   mul_comm f g := by ext x; simp [mul_apply, mul_comm]
 
+/-- Coercion of a `BoundedContinuousFunction` is an `MonoidHom`. Similar to `MonoidHom.coeFn`. -/
+@[to_additive (attr := simps) /-- Coercion of a `BoundedContinuousFunction` is an `AddMonoidHom`.
+Similar to `AddMonoidHom.coeFn`. -/]
+def coeFnMulHom [Monoid R] [BoundedMul R] [ContinuousMul R] : (α →ᵇ R) →* α → R where
```

Comment:
While you're at it, could you rename this to `coeFnMonoidHom`? See e.g [ContinuousMap.coeFnMonoidHom](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/ContinuousMap/Algebra.html#ContinuousMap.coeFnMonoidHom).